src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java

```diff
@@ -1786,10 +1807,32 @@ protected String extractAuthorName(Document doc) {
                 }
             }
 
+            // globalbankingandfinance.com
+            if (authorName.isEmpty()) {
+                authorName = SHelper.innerTrim(doc.select("div[class=the-content post-content clearfix] p strong em").text());
+                if(DEBUG_AUTHOR_EXTRACTION && !authorName.isEmpty()) System.out.println("AUTHOR: div[class=the-content post-content clearfix] p strong em");
+            }
+
+            // fortune.com
+            if (authorName.isEmpty()) {
+                authorName = SHelper.innerTrim(doc.select("head meta[property=author]").attr("content"));
+                if(DEBUG_AUTHOR_EXTRACTION && !authorName.isEmpty()) System.out.println("AUTHOR: head meta[property=article:author]");
+            }
+
             if (authorName.isEmpty()) {  // for "opengraph"
```

Comment:
Keeping general cases at top doesn't work well for all the websites. e.g. For huffingtonpost.com if general case is at top it will not give us the accurate result so sequence need to be adjusted.